app/views/static_contents/_form.html.erb

```diff
@@ -27,7 +27,20 @@
 
   <div class="form-group">
     <%= form.label :content %>
-    <div id="code_editor" style="height: 400px;"></div>
+    <% if static_content.data_type == "json" %>
+      <% content = static_content.content.to_s.html_safe %>
+      <% valid_json_content = JSON.parse(content) rescue nil %>
+      <% if valid_json_content %>
+        <div id="code_editor" style="height: 400px;"></div>
+      <% else %>
+        <div>
+          <p class="text-danger"><small><em>JSON format is not valid!</em></small></p>
```

Comment:
i decided to take an english info text here, because the labels in show/edit views are english as well. but i do not like the english/german mixture in the main server views in general :(